src/Infrastructure/Controller/Regulation/Fragments/GetLocationController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller\Regulation\Fragments;
+
+use App\Application\Regulation\View\Measure\MeasureView;
+use App\Domain\Regulation\Repository\LocationRepositoryInterface;
+use App\Domain\Regulation\Repository\MeasureRepositoryInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\Requirement\Requirement;
+
+final class GetLocationController extends AbstractController
+{
+    public function __construct(
+        private \Twig\Environment $twig,
+        private LocationRepositoryInterface $locationRepository,
+        private MeasureRepositoryInterface $measureRepository,
+    ) {
+    }
+
+    #[Route(
+        '/_location/{uuid}',
+        name: 'get_location',
+        requirements: ['uuid' => Requirement::UUID],
+        methods: ['GET'],
+    )]
+    public function __invoke(Request $request, string $uuid = ''): Response
+    {
+        $location = null;
+        $measureAsAView = null;
+        $regulation = null;
+        $regulationOrderRecordId = null;
+        if ($uuid) { // uuid of a location
+            $location = $this->locationRepository->findOneByUuid($uuid);
+            if ($location) {
+                $measure = $this->measureRepository->findOneByUuid($location->getMeasure()->getUuid());
```

Comment:
L'appel à ce repository n'est pas nécessaire. Si `$this->locationRepository->findOneByUuid($uuid)` te renvoie un objet `Location`, tu peux directement faire `$location->getMeasure()`. Ca t'évite donc une requête supplémentaire.